proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -449,11 +449,34 @@ message PowerShapingParameters {
   // have to validate the proposed consumer chain. top_N can either be 0 or any value in [50, 100].
   // A chain can join with top_N == 0 as an Opt In chain, or with top_N ∈ [50, 100] as a Top N chain.
   uint32 top_N = 1;
-  // Corresponds to the maximum power (percentage-wise) a validator can have on the consumer chain. For instance, if
-  // `validators_power_cap` is set to 32, it means that no validator can have more than 32% of the voting power on the
-  // consumer chain. Note that this might not be feasible. For example, think of a consumer chain with only
-  // 5 validators and with `validators_power_cap` set to 10%. In such a scenario, at least one validator would need
-  // to have more than 20% of the total voting power. Therefore, `validators_power_cap` operates on a best-effort basis.
+  // `validators_power_cap` corresponds to the maximum power (percentage-wise) a validator can have on the consumer chain.
```

Comment:
People seemed to have difficulty understanding the validators power cap, so adding more examples here to make it more accessible. (cc @ve6a)